estmator_project/est_client/models.py

```diff
@@ -55,3 +55,20 @@ class Client(models.Model):
 
     def __str__(self):
         return self.first_name + " " + self.last_name
+
+    def quotes_select_html(self):
+        """Return HTML of select elements for all client quotes."""
+        quotes = self.quotes.all()
+        quote_form = '<p>\n' \
+                     '<label for="id_quote">Quote:</label>' \
+                     '<select id="id_quote" name="quote">\n' \
+                     '<option value selected="selected">---------</option>\n'
+        for quote in quotes:
+            quote_form += (
+                '<option value="{id}">{name}: {date}</option>\n'
+                .format(
+                    id=quote.id, name=quote.name, date=quote.date
+                )
+            )
+        quote_form += '</select></p>\n'
+        return quote_form
```

Comment:
hmmm.  It feels to me like this should be taken care of by something like the `ModelSelectField` or `ModelMultiSelectField` which should provide you with a form widget that renders all the instances of a given model, like `Quote` as a series of options in a select field.  It manages things like the ids of the individual options for you.  Did you find and discount this, or miss finding it in the Forest of Django Options?